build/docker/docker-compose.yml

```diff
@@ -48,13 +53,24 @@ services:
     container_name: elasticsearch
     hostname: elasticsearch
     ports:
-      - "19200:9200"
-    command: ["elasticsearch"]
+      - "9200:9200"
+    command: [ "elasticsearch" ]
     networks:
       default:
         aliases:
           - elasticsearch.sfs.local
-
+  mockservice:
+    image: rodolpheche/wiremock
+    container_name: mockservice
+    volumes:
+      - $PROJECT_DIR/test/wiremock_mappings:/home/wiremock/mappings/
+    ports:
+      - "8090:8080"
+    command: [ "--verbose"]
+    networks:
+      default:
+        aliases:
+          - mockservice.sfs.test
```

Comment:
Is it possible to rename this URL so it's clearer what it is. Something like: mock.postcodeapi.sfs:8080/api ?